ui/rla_ui.js

```diff
@@ -1014,4 +1025,253 @@ function restrictInput(event) {
    return pattern.test(value);
 }
    
+// JSON File upload functions HERE
+function maybeGetJSONFile(ballotType) {
+    return function() {
+        document.body.appendChild(jsonFileUploadContainer);
+        if (conductorState['json_hash'][ballotType] === undefined) {
+            uploadJSONFile(ballotType);
+        } else {
+            displayJSONFile(ballotType);
+            mainLoop();
+        }
+    }
+};
+
+ // JSON Function
+function uploadBallotManifestJson(ballotType) {
+  makeBallotManifestUploadContainerJson(ballotType);
+};
+ // Creates front end form Container for JSON uploading
+function makeBallotManifestUploadContainerJson(ballotType) {
```

Comment:
This is pretty similar to `makeBallotManifestUploadContainer` - can we make a general function that works for all three uploads, and takes as arguments any differences?